compiler/esy.json

```diff
@@ -20,7 +20,7 @@
     "check-format": "dune build @fmt"
   },
   "dependencies": {
-    "@grain/binaryen.ml": "0.8.0",
+    "@grain/binaryen.ml": "0.8.1",
```

Comment:
Whoops, this will need to be removed now that I merged 0.9.0 bump. I suppose you'll rebase this on after #551 is merged anyway.